lib/ModelMixins/GroupMixin.ts

```diff
@@ -49,6 +49,24 @@ function GroupMixin<T extends Constructor<Model<GroupTraits>>>(Base: T) {
       );
     }
 
+    @action
+    sortMembers(
+      stratumId: string,
+      sortFunction: (a: BaseModel, b: BaseModel) => number
+    ) {
+      const members: any = this.getTrait(stratumId, "members");
+      const models = this.memberModels as Array<BaseModel>;
+      if (members === undefined || models.length === 0) return;
+      const unsortedModels = models.splice(0);
+      members.clear();
```

Comment:
Not sure why `this.getTrait(stratumId, "members")` isn't giving correct type with the `clear` method on it - I'll have a look into it

Reply:
Yeah that annoyed me too, hence going the `any` approach.